src/ttt_core/console.clj

```diff
@@ -59,50 +65,73 @@
 (defn draw-main [game message]
   (do
     (clear-terminal)
-    (draw-header)
+    (draw-header (:language game))
     (draw-player-info game)
     (draw-board (:board game))
     (println message)
     (draw-footer)))
 
-(defn build-choose-game-mode-string 
-  ([] (build-choose-game-mode-string ""))
-  ([message]
-    (str message "Choose a game mode:\n1. Human-vs-Human\n2. Human-vs-Computer")))
-
-(defn request-game-mode
-  ([game] (request-game-mode game ""))
-  ([game message]
-    (let [modes {:1 :human-vs-human :2 :human-vs-computer}]
+(defn build-selection-string 
```

Comment:
`build-selection-string` function supports multiple strings (drawn from the translate dictionary) with prepended messages. The purpose was to DRY up the code rather than having separate build functions like `build-game-mode-selection-string` and `build-language-selection-string` etc.